The Great IMU War

mjmagee991 opened the pull request:
Cleaned up and implemented IMU main
Sped up CANOpen output

Commit:
Update BNO and IMU samples for bringups

Commit:
Updated docs

Commit:
Finished off formatting and testing with the breakout board

mjmagee991 committed:
Updated EVT-core

mjmagee991 committed:
Cleaned Up IMU Main

Commit:
Applied Formatting Changes During GitHub Build

ActuallyTaylor commented on src/IMU.cpp:
```diff
@@ -40,7 +38,7 @@ void IMU::process() {
 
     // Retrieve the Accelerometer X, Y, and Z values from the bno055
-    bno055.getAccelerometer(vectorXValues[3], vectorYValues[3], vectorZValues[3]);
+    bno055.getLinearAccel(vectorXValues[3], vectorYValues[3], vectorZValues[3]);
```
Why was this changed to linear accel?

mjmagee991 committed:
PR Cleanup

Changed name of I2C Address variable
Fixed IMU.cpp Typo
Fixed IMU.hpp Documentation

mjmagee991 committed:
Merge remote-tracking branch 'origin/imu-bringup' into imu-bringup
